client/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "esModuleInterop": true,
     "target": "es6",
     "allowJs": true,
-    "lib": ["es6", "dom", "esnext"]
+    "lib": ["es6", "dom", "esnext"],
+    "baseUrl": "."
```

Comment:
nit: you could use `./src` instead of `.` since all your code is located in `src`

Reply:
That is what I thought in the first place too ; however I think it is great to ship components with test files close to other files (such as in `./src/views/Mapping/components/FhirMappingPanel/components/Drawer`, where `index.tsx` and `index.test.tsx` are in the same folder), therefore I also need to access files which are stored under `./test`.